istio-proxy/istio-proxy.spec

```diff
@@ -16,7 +16,7 @@
 %global git_commit a405165e3e09e57217b53846454db913cd493d10
 %global shortcommit  %(c=%{git_commit}; echo ${c:0:7})
 %global gn_commit 743970903a39f4122a6476f9e05f59b9af03fdf6
-%global gn_hash 259564bf3c706413a7cdadf4c8cd6a49
+%global gn_hash 95309ab14cfdd6bf8e129d20dd25b43a
```

Comment:
Why did GN change?

Reply:
To match the product version

Reply:
Right, I guess my next question is, why did it change in product side? :)